examples/gno.land/r/gov/dao/v3/memberstore/types.gno

```diff
@@ -123,7 +123,9 @@ func (mbt MembersByTier) RemoveMember(addr std.Address) (t string) {
 		}
 
 		_, removed := mst.Remove(string(addr))
-		t = tn
+		if removed {
+			t = tn
+		}
```

Comment:
it's a simple fix, but I believe this change should have tests added

Reply:
indeed, should get the reflex to write more tests, thx for feedback 
[d30d8e6](https://github.com/gnolang/gno/pull/4473/commits/d30d8e6f3937691d12b9ffcd99cc82b60dcd8431)